vendor/index.yaml

```diff
@@ -1691,7 +1691,7 @@ vendors:
     website: https://www.innotas-elektronik.de/en-gb
 
   - id: elsys
-    name: ElektronikSystem i Ume
+    name: Elsys
```

Comment:
Why are they in here twice @Jaime-Trinidad @johanstokking ? Can't we delete the second?

Reply:
Yes that is wrong. This one can and should be removed